examples/graphene_apps/python_worker/avalon_worker/work_order_processor.py

```diff
@@ -119,6 +110,9 @@ def _generate_worker_signup(self):
             self.worker_public_enc_key.decode("utf-8")
         signup_data_json["encryption_key_signature"] = \
             self.worker_public_enc_key_sign.hex()
+        # Create Graphene SGX Attestation instance.
+        enclave_type = SgxAttestationFactory.GRAPHENE
```

Comment:
Pass directly if not using the variable again. There are languages which will make you do so in static analysis. :)

Reply:
Make sense